src/signals/settings/categories/components/Detail.test.tsx

```diff
@@ -1,49 +1,38 @@
 // SPDX-License-Identifier: MPL-2.0
 // Copyright (C) 2023 Gemeente Amsterdam
-import { render, screen } from '@testing-library/react'
+import { render, screen, waitFor } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import * as reactRedux from 'react-redux'
 import { MemoryRouter } from 'react-router-dom'
 import * as reactRouterDom from 'react-router-dom'
 
 import { withContext } from 'components/Summary/Summary.test'
-import useFetch from 'hooks/useFetch'
-import { useFetchResponse } from 'signals/IncidentMap/components/__test__/utils'
 import { subCategories } from 'utils/__tests__/fixtures'
 import historyJSON from 'utils/__tests__/fixtures/history.json'
 
 import { CategoryDetail } from './Detail'
 import type { Props } from './Detail'
+import * as API from '../../../../../internals/testing/api'
+import {
+  fetchMock,
+  mockRequestHandler,
+} from '../../../../../internals/testing/msw-server'
 
-jest.mock('hooks/useFetch')
+fetchMock.disableMocks()
```

Comment:
not in beforeAll block?

Reply:
This is how they implement it everywhere